cook-book/getting-started/README.md

```diff
@@ -0,0 +1,287 @@
+# Getting Started
+
+This simple guide shows how to deploy Tarantool Cartridge application
+and set up topology.
+
+First, we will deploy application package to virtual machines, start
+two instances and join them to replicaset.
+Then, we will start one more instance and create replicaset
+step-by-step.
+
+## Requirements
+
+* `Cartridge CLI`
+* `tarantool.cartridge` role >= `1.8.2`
+
+Check out the [configuration basics guide](/doc/configuration-basics.md)
+to be sure that you understand the way to write cluster configuration.
+
+## Prepare
+
+Install role from Ansible Galaxy (use newer version if it's available):
+
+```bash
+$ ansible-galaxy install tarantool.cartridge,1.8.2
+```
+
+Start two virtual machines described in [Vagrantfile](./Vagrantfile):
+
+```bash
+vagrant up
+```
+
+Place [application package](/cook-book/create-packages.md) into current directory or specify right path in `hosts.yml`.
+
+## Deploy application and set up topology
+
+First, deploy first version of application on virtual machines,
+start two instances and join them to replicaset.
+This configuration is now
+
+Run playbook:
+
+```bash
+ansible-playbook -i hosts.yml playbook.prepare.yml
+```
+
+Now go to http://localhost:8182/admin/cluster/dashboard:
+
+![image](./images/initial.png)
+
+### Using scenario
+
+We've just start two instances and join them to replicaset via running one playbook.
+But this is bad approach to start instances, edit topology and configure cluster
+at once.
+We will start more instances, add them to topology and edit cluster
+configuration step by step.
+
+Since version [1.8.0](https://github.com/tarantool/ansible-cartridge/releases/tag/1.8.0)
+`tarantool.cartridge` role allows to specify [scenario](/doc/scenario.md)
+that consists of steps.
+The role has pre-defined [steps](/doc/scenario.md#steps), but you can also
+[write your own step](/doc/scenario.md#adding-custom-step-to-scenario) and
+use it in scenario.
+
+Using scenarios allows to perform different actions with cluster separately.
+
+The role has several common [scenarios](/doc/scenario.md#scenarios) rhat
+can be re-used.
+It's also possible to write custom scenario and then use it in different plays.
+
+### Change configuration
+
+Now we have such topology:
+
+* `storage-1` replicaset (`vshard-storage`):
+  * `storage-1-leader`
+  * `storage-1-replica`
+
+Let's add `core-1` replicaset and add one more replica to `storage-1` replicaset:
+
+* `core-1` replicaset (`vshard-router`, `failover-coordinator`):
+  * `core-1`
```

Comment:
It's not good that replica set and instance names are the same

Reply:
I don't have any ideas =(

Reply:
core-1-replicaset